From 2d44b2824789e05abc2507188a9b309ced82dfbf Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Fri, 10 Sep 2021 10:23:35 +0300
Subject: [PATCH] [SW-0] gaudi2: pll: add comment for sram

Add comment stating the frequency we configure the PLL DIV output 0 to.
No functional changes.

Change-Id: If5d1964a1e7a55f55482a598d06fd4604640a213
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/arch/gaudi2/common/asic.c b/arch/gaudi2/common/asic.c
index d42b3f3..81ada1c 100644
--- a/arch/gaudi2/common/asic.c
+++ b/arch/gaudi2/common/asic.c
@@ -77,7 +77,7 @@
 	}, {
 		.name = "DCORE2_SRAM",
 		.base = mmDCORE2_XBAR_BANK_PLL_CTRL_BASE,
-		.nr = 1, .nf = 64, .od = 2, .nb = 4,
+		.nr = 1, .nf = 64, .od = 2, .nb = 4,	/* 400 */
 		.div_factor = { 0, -1, -1, -1 },
 		.div_sel = { HL_PLL_PLL_CLK,
 			     HL_PLL_REF_CLK,
@@ -87,7 +87,7 @@
 	}, {
 		.name = "DCORE3_SRAM",
 		.base = mmDCORE3_XBAR_BANK_PLL_CTRL_BASE,
-		.nr = 1, .nf = 64, .od = 2, .nb = 4,
+		.nr = 1, .nf = 64, .od = 2, .nb = 4,	/* 400 */
 		.div_factor = { 0, -1, -1, -1 },
 		.div_sel = { HL_PLL_PLL_CLK,
 			     HL_PLL_REF_CLK,
